src/discussions/posts/post/PostFooter.jsx

```diff
@@ -62,6 +62,9 @@ function PostFooter({
             </span>
           </>
         )}
+      {post.unreadCommentCount && post.unreadCommentCount > 0 && post.commentCount > 1 ? (
+        <Badge variant="light">{post.unreadCommentCount} {intl.formatMessage(messages.newLabel)}</Badge>
```

Comment:
I think it might be better to include the number with the message so that translators can potentially change the language based on pluralisation, i.e. make this:
```suggestion
        <Badge variant="light">{intl.formatMessage(messages.newLabel, {count: post.unreadCommentCount})}</Badge>
```

and update the message to be:
```
defaultMessage: '{count} new',
```

Reply:
@tecoholic ^ in case you weren't pinged.

Reply:
@xitij2000 Thanks for this. I didn't know of this usage. Didn't read the docs deep enough I guess. :/